static/styles/settings.css

```diff
@@ -892,13 +898,13 @@ input[type="checkbox"] {
 
 .edit_bot_email {
     font-weight: 400;
-    font-size: 18px;
+    font-size: 15px;
     text-align: left;
     margin-top: 0;
-    margin-bottom: 10px;
+    margin-bottom: 15px;
 
     overflow: hidden;
-    max-height: 1.1em;
+    max-height: 1.4em;
```

Comment:
Why this change? I think you can just remove `max-height` from here.

Reply:
it was already there, 1.1 was making p of zulip cut out so I just increased it. 
but I will just remove it